pulser/register.py

```diff
@@ -18,11 +18,12 @@
 from abc import ABC, abstractmethod
 from collections.abc import Mapping, Iterable
 from collections.abc import Sequence as abcSequence
-import matplotlib.pyplot as plt
+
 from matplotlib import collections as mc
 import numpy as np
 from numpy.typing import ArrayLike
 from scipy.spatial import KDTree
+import matplotlib.pyplot as plt
 from typing import Any, cast, Optional, Union, TypeVar, Type
 from itertools import combinations
```

Comment:
Hum, these shouldn't be here... I know it's not your fault, but could you please put them on the first block while you're at it?